spec/features/courses/description_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+
+feature 'Course description', type: :feature do
+  let(:provider) { jsonapi(:provider, accredited_body?: false) }
+  let(:course) {
+    jsonapi :course,
+      funding: 'fee',
+      site_statuses: [site_status],
+      provider: provider,
+      accrediting_provider: provider
+  }
+  let(:site) { jsonapi(:site) }
+  let(:site_status) do
+    jsonapi(:site_status, :full_time_and_part_time, site: site)
+  end
+  let(:course_response) { course.render }
+  before do
+    stub_omniauth
+    stub_session_create
+    stub_api_v2_request(
+      "/providers/A0/courses/#{course.attributes[:course_code]}?include=site_statuses.site,provider.sites,accrediting_provider",
+      course_response
+    )
+  end
+
+  let(:course_page) { PageObjects::Page::Organisations::CourseDescription.new }
+
+  describe 'with a fee paying course' do
+    scenario 'it shows the course description page' do
+      visit "/organisations/A0/courses/#{course.attributes[:course_code]}/description"
+
+      expect(course_page.caption).to have_content(
+        course.attributes[:description]
```

Comment:
This would read so nicely if we could do `course.description` instead of this attributes nonsense. :(

Reply:
Why can't we?